ui/src/app/store/machine/utils/networking.ts

```diff
@@ -614,3 +614,34 @@ export const getRemoveTypeText = (
     return interfaceType;
   }
 };
+
+/**
+ * Find the next available name for an interface.
+ * @param machine - A machine.
+ * @param interfaceType - A network interface type.
+ * @return An available name.
+ */
+export const getNextNicName = (
```

Comment:
Awesome! I did the same thing but in like 6 different places in the storage tab - maybe if we get some time at the end of the cycle we should align a lot of these things.

Reply:
If you remember where you did it would you mind filing an issue? I just had a quick look and didn't spot them.

Reply:
Done

https://github.com/canonical-web-and-design/maas-ui/issues/2188